client/my-sites/stats/activity-log/threat-alert.jsx

```diff
@@ -35,38 +29,58 @@ export class ThreatAlert extends Component {
 						<div className="activity-log__threat-alert-header">
 							<div>
 								<span className="activity-log__threat-alert-title">
-									{ translate( 'Thread found' ) }
+									{ threat.diff && translate( 'Threat found in core WordPress file' ) }
```

Comment:
Yeah we could but this is pretty much the extend of it. Is it mainly the translations? Because we could break out the transition into a separate function easily enough I think. Anything more and I'm worried it will get too repetitive. Let me see what I can do with translations…

Thanks!